modules/tinymce/src/themes/silver/test/ts/webdriver/editor/SimpleControlsInlineTest.ts

```diff
@@ -0,0 +1,32 @@
+import { RealMouse, UiFinder, Waiter } from '@ephox/agar';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { McEditor } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.editor.core.SimpleControlsInlineTest', () => {
+  before(() => Theme());
```

Comment:
```suggestion
  before(() => {
    Theme()
  });
```
We should never use shorthand for hooks, as the return value from the function executed will be passed to the hook.